packs/apache/config.yml

```diff
@@ -7,33 +7,19 @@ name: Apache
 description: |+
   # Complete Apache Monitoring Kit
```

Comment:
```suggestion
  ## Complete Apache Monitoring Kit
```

Reply:
@bradleycamacho if you can accept this suggestion, this looks good!

Reply:
I couldn't accept this suggestion since it was outdated, but I pushed up a commit implementing it.